pocs/scheduler/scheduler.py

```diff
@@ -59,6 +60,7 @@ def __init__(self, observer, fields_list=None, fields_file=None,
             self.logger.debug("Reading initial set of fields")
             self.read_field_list()
 
+        self.common_properties = None
```

Comment:
```suggestion
        # Items common to each observation that shouldn't be computed each time.
        self.common_properties = None
```

Add a comment describing what this is.